test-basic/unnestTest.js

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2023 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const assert = require('assert');
+const marklogic = require('../');
+const testconfig = require('../etc/test-config.js');
+const db = marklogic.createDatabaseClient(testconfig.restWriterConnection);
+const op = marklogic.planBuilder;
+
+const TEAM_MEMBER_NAME_COLUMN = "teamMemberName";
+
+
+describe('test unnest function', function () {
+
+    before(function (done) {
+
+        db.documents.remove(['/optic/test/office1.json'])
```

Comment:
I like this, and what I recommend trying to do later on (not in this PR) is adding a "delete everything that ought to be deleted" function that every test uses (ideally, this can be done in one place in the test code so it doesn't have to be repeated in every test).